bin/sozo/src/commands/clean.rs

```diff
@@ -41,18 +35,96 @@ impl CleanArgs {
         let profile_name =
             ws.current_profile().expect("Scarb profile is expected at this point.").to_string();
 
-        let clean_manifests_abis = self.manifests_abis || !self.artifacts;
-        let clean_artifacts = self.artifacts || !self.manifests_abis;
+        let manifest_dir = ws.manifest_path().parent().unwrap().to_path_buf();
 
-        if clean_manifests_abis {
-            let manifest_dir = ws.manifest_path().parent().unwrap().to_path_buf();
-            self.clean_manifests_abis(&manifest_dir, &profile_name)?;
-        }
+        let profile_dir = manifest_dir.join(MANIFESTS_DIR).join(profile_name);
 
-        if clean_artifacts {
-            scarb::ops::clean(config)?;
+        // By default, clean build manifests and scarb artifacts.
+        scarb::ops::clean(config)?;
+        self.clean_manifests(&profile_dir)?;
+
+        if self.all && profile_dir.exists() {
+            fs::remove_dir_all(profile_dir)?;
```

Comment:
any reason why we don't just remove the `/manifests` entirely instead of just the profile dir?

Reply:
As we're now based on the profile, we may only want to clean the active profile. Or do you think we should remove everything? It looked a bit harsh to be to clean up everything as some profile may be used only for production and we only messed up with your dev profile.

Reply:
ah make sense... but how does one specify which profile to clean ? there doesn't seem to be a flag to specify that, or will that be a follow up?